rnn_translator/pytorch/train.py

```diff
@@ -259,13 +254,16 @@ def main():
     """
     Launches data-parallel multi-gpu training.
     """
-    mlperf_log.ROOT_DIR_GNMT = os.path.dirname(os.path.abspath(__file__))
-    mlperf_log.LOGGER.propagate = False
+    _file_handler = logging.FileHandler(os.path.join(os.path.dirname(os.path.abspath(__file__)), 'gnmt.log'))
```

Comment:
FYI, I added a convenience option that allows adding a log file in one line (the file set in this way will be using a default config). Here is the updated example https://github.com/xyhuang/training/blob/mllog/compliance/mllog/examples/dummy_example.py#L49
The current implementation here should still work, though it might be a bit more recommended to use the mllog.config() function (see the same example file) so people don't have to understand internals of the mllogger.

Reply:
Done here:
https://github.com/mlperf/training/blob/44180125b54c1d244299850ced80e6abf2b7161c/rnn_translator/pytorch/train.py#L257